autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,135 @@
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}"
+sysrc ifconfig_${ifdev}=DHCP
+sleep 5
+
+# Set latest repository to the new system
+echo "Set latest repository to the new system ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -f | cat
+bit=`uname -m`
+OSFamily=`uname -s`
+major_version=`freebsd-version -k | cut -d . -f 1`
+repo="${OSFamily}:${major_version}:${bit}"
+
+mkdir -p /usr/local/etc/pkg/repos
+cat << EOF > /usr/local/etc/pkg/repos/latest.conf
+FreeBSD: { enabled: no }
+
+latest: {                                                                     
+  url: "pkg+http://pkg.FreeBSD.org/${repo}/latest",
+  mirror_type: "srv",                                                           
+  signature_type: "fingerprints",                                               
+  fingerprints: "/usr/share/keys/pkg",                                         
+  enabled: yes                                                                 
+}
+EOF
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
+
+# Installing packages
+echo "Installing packages ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -y
+env ASSUME_ALWAYS_YES=YES pkg update -f
+# Hit issue: reset by peer during install packages
+# The open-vm-tools is not installed by default
+packages_to_install='python3 open-vm-tools-nox11 sudo'
+for package_to_install in $packages_to_install
+do
+    echo aaa | grep bbb
+    until [ $? -eq 0 ]
+    do
+        echo "will install package: $package_to_install" > /dev/ttyu0
+        check_results=`env ASSUME_ALWAYS_YES=YES pkg install -y $package_to_install`
+        echo $check_results > /dev/ttyu0
+        echo $check_results | grep "are already installed"
+    done
+    echo "The package($package_to_install) is already installed" > /dev/ttyu0
+done
+# env ASSUME_ALWAYS_YES=YES pkg install -y sudo lsblk bash curl wget sudo
+
+# Add new user. 
+{% if new_user is defined and new_user != 'root' %}
+echo "{{ vm_password }}" | pw -V /etc useradd {{ new_user }} -h 0 -s /bin/sh -G wheel -d /home/{{ new_user }} -c "{{ new_user }} User"
+mkdir -p /home/{{ new_user }}
+chown 1001:1001 /home/{{ new_user }}
+echo '$new_user ALL=(ALL:ALL) ALL' >> /usr/local/etc/sudoers
+{% endif %}
+
+# Set password of root user
+echo "{{ vm_password }}" | pw -V /etc usermod root -h 0
+
+# Enable root login via ssh
+echo "Enable root login via ssh ..." > /dev/ttyu0
+mkdir -p -m 700 /root/.ssh
+echo "{{ ssh_public_key }}" > /root/.ssh/authorized_keys
+chown -R root /root/.ssh
+chmod 0644 /root/.ssh/authorized_keys
+# We can't ssh to VM with empty password for root user
+sed -i .bak -e 's/#PasswordAuthentication.*/PasswordAuthentication yes/' /etc/ssh/sshd_config
+sed -i '' -e 's/#PermitRootLogin.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+sed -i '' -e 's/#PermitEmpty.*/PermitEmptyPasswords yes/' /etc/ssh/sshd_config
```

Comment:
Shall `yes` be `no` for `PermitEmptyPasswords`?

Reply:
Change it. Thanks for your suggestion.